contrib/remote-dev/README.md

```diff
@@ -0,0 +1,165 @@
+```
+
+After configuration, you should edit the job template file named `./conf/job.template`.
+
+The original job template file is like:
+
+```yaml
+protocolVersion: 2
+name: remote_dev_job_template
+type: job
+jobRetryCount: 0
+prerequisites:
+  - type: dockerimage
+    uri: openpai/pai.example.tensorflow
+    name: docker_image_0
+taskRoles:
+  Task_role_1:
+    instances: 1
+    completion:
+      minFailedInstances: 1
+      minSucceededInstances: 1
+    dockerImage: docker_image_0
+    resourcePerInstance:
+      gpu: 0
+      cpu: 1
+      memoryMB: 2048
+    commands:
+      - '`#REMOTE_DEV_START`'
+      - apt-get update
+      - apt-get install -y nfs-common cifs-utils sshpass wget
+      - umask 000
+      - mkdir -p /root/workspace
+      - mount.cifs //sharepath /root/workspace -o username=***,password=***
+      - sleep 18000
+      - '`#REMOTE_DEV_END`'
+    taskRetryCount: 0
+defaults:
+  virtualCluster: default
+```
+
+Users should edit the resources in `resourcePerInstance`, and can also change docker image in `resourcePerInstance`. Please make sure your customized docker image has openssh-server. Users can also change `commands` if they want to mount storage or install dependencies in container.
+
+We suppose that your training data is stored in storage server and managed by PAI's [Team Wise Storage](https://github.com/microsoft/pai/tree/master/contrib/storage_plugin). You should change `commands` to mount your data server.
+
+The life cycle of this container is 18000s (5 hours). Considering that this container is for development and debugging, it is not recommended to run for a long time. If you have a specific usage, you can change `sleep 18000` to `sleep infinity` and **don't forget to manually stop this job**.
+
+For more details about job template, please refer to [PAI Job Protocol](https://github.com/microsoft/pai/blob/master/docs/pai-job-protocol.yaml).
+
+### 3.2 Ask PAI for Resources
+
+After preparation, you can run `start.py` to ask pai for a container with resources.
+
+```sh
+python3 start.py
+```
+It is important to note that it takes some time to wait for the job to run. You can check job status from PAI's web portal. If no error occurs, you will get ssh info like this:
+
+![](./doc/sshinfo.png)
+
+The ssh info includes `SSH IP`, `SSH Port`, `SSH Key`, and `SSH CMD`. You can ssh into your container with the above `SSH CMD`. If you are using windows, your ssh key will be stored in `%UserProfile/.openpai/***.key`. If you are using ubuntu, your ssh key will be stored in `~/.openpai/***.key`. Please take care of these keys.
+
+### 3.3 Configure Editors and IDEs
+
+#### 3.3.1 CLI
+
+For CLI usage, just ssh into your container and you can do anything you want. Your workspace will be mounted at `/root/workspace`.
+
+#### 3.3.2 PyCharm Professional
+
+For PyCharm Professional usage, you can configure PyCharm `Deployment` and `Project Interpreter` according to the following animation.
```

Comment:
Please clarify that it didn't work for community version

Reply:
This can only work in professional version. Will make it more clear.